presets/party/README.md

```
@@ -0,0 +1,117 @@
+[![banner](https://particles.js.org/images/banner3.png)](https://particles.js.org)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)